fairing/functions/function_shim.py

```diff
@@ -0,0 +1,17 @@
+import argparse
+from importlib import import_module
+
+
+def call(module_name, class_name, function_name):
+    mod = import_module(module_name, function_name)
```

Comment:
In the example, train function is defined in the __main__ module. How does this work if function_shim is the __main__ module when it is executed remotely.

Reply:
function_shim calls the function directly, if that makes sense

Reply:
what will be the values of module_name and function_name in the example given here?

Reply:
```
    Command:
      python
      /app/function_shim.py
      --module_name
      main
      --function_name
      train
```

Reply:
looks good.